src/lib/models/reflections/project.ts

```diff
@@ -123,25 +117,4 @@ export class ProjectReflection extends ContainerReflection {
 
         return undefined;
     }
-
-    /**
-     * Return a raw object representation of this reflection.
-     * @deprecated Use serializers instead
-     */
-    toObject(): any {
```

Comment:
Any reason this was deleted? The `.toObject` methods are still used in tests right now.

Reply:
ProjectReflection extends ContainerReflection, which already outputs categories in its toObject method.